README.md

```diff
@@ -1 +1,44 @@
-# selected-tests
+# Selected Tests
+
+This service is used to predict which tests need to run based on code changes.
+
+# Set up environment
```

Comment:
I considered adding this, but we don't mention virtual environments in our other python READMEs, I think virtual environments are assumed

Reply:
Using a virtualenv isn't a requirement. It is a really good idea to keep dependencies separate, but if a developer really want to manage environments in a different way they could. We should probably have something on the wiki that talks about using virtualenv because it would be the same for all projects.